docs/proposals/systemd-nodespec.md

```diff
@@ -0,0 +1,134 @@
+# Kubernetes Node Specification - systemd
+
+**Author**: Derek Carr (@derekwaynecarr)
+
+**Status**: Proposed
+
+*This document presents a specification for how the ```kubelet``` interfaces with ```systemd``` with a focus on quality of service concerns.*
+
+## Motivation
+
+Linux distributions have either adopted, or plan to adopt ```systemd``` as their init system.
+
+This document describes a series of enhancements that should be made to the ```kubelet``` to better integrate with these distributions.
+
+It has a primary focus on describing how the ```kubelet``` interaction with ```cgroups``` should change to work through ```systemd```.
```

Comment:
I think this should be part of the last paragraph if you're going to use 'It' unqualified in the beginning of this sentence.